### PR TITLE
Add ARM v7 targets to CI

### PR DESCRIPTION
**Pull request by jfalcou**

### CODE REVIEW

File: test/unit/meta/has_abi.cpp [hunk 1]
```diff
@@ -15,34 +15,38 @@
...
-  TTS_EXPECT( eve::has_native_abi_v<eve::wide<double>>                );
-  TTS_EXPECT( eve::has_native_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT( eve::has_native_abi_v<eve::wide<float>>                );
+  TTS_EXPECT( eve::has_native_abi_v<eve::logical<eve::wide<float>>>  );
 #endif
 
-  TTS_EXPECT( eve::has_native_abi_v<double>                           );
-  TTS_EXPECT( eve::has_native_abi_v<eve::logical<double>>             );
+  TTS_EXPECT( eve::has_native_abi_v<float>                           );
+  TTS_EXPECT( eve::has_native_abi_v<eve::logical<float>>             );
```
Comment (DenisYaroshevskiy) [inline test/unit/meta/has_abi.cpp:32]: Eeh? This seems like a duplicate

File: test/unit/meta/has_abi.cpp [hunk 2]
```diff
@@ -15,34 +15,38 @@
...
 
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::wide<double>>                );
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::wide<float>>                );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<eve::wide<float>>>  );
 
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<double>                           );
-  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<double>>             );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<float>                           );
+  TTS_EXPECT_NOT( eve::has_aggregated_abi_v<eve::logical<float>>             );
```
Comment (DenisYaroshevskiy) [inline test/unit/meta/has_abi.cpp:51]: No idea what this is

File: test/unit/meta/has_abi.cpp (hunk 1 above)
Comment (jfalcou) [reply test/unit/meta/has_abi.cpp:32]: We check that scalar have indeed a native ABI.

File: test/unit/meta/has_abi.cpp (hunk 2 above)
Comment (jfalcou) [reply test/unit/meta/has_abi.cpp:51]: We check scalars don't get assigned the aggregated ABI
Those are old tests, but they check out ;)

**Review by DenisYaroshevskiy — APPROVED** [review]

**Pull request merged**